cpp/src/knn/knn_mg.cu

```diff
@@ -1,365 +1,84 @@
 /*
- * Copyright (c) 2019-2020, NVIDIA CORPORATION.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#include <cuml/neighbors/knn_mg.hpp>
-#include <selection/knn.cuh>
-
-#include <common/cumlHandle.hpp>
-
-#include <common/device_buffer.hpp>
-#include <cuml/common/cuml_allocator.hpp>
-#include <raft/comms/comms.hpp>
-
-#include <set>
-
-#include <raft/cudart_utils.h>
-#include <raft/cuda_utils.cuh>
+* Copyright 1993-2020 NVIDIA Corporation.  All rights reserved.
```

Comment:
Any files touched / created in this PR should now be 2021.

Reply:
Also this license should remain the Apache license header